meshnets/utils/model_loading.py

```diff
@@ -0,0 +1,51 @@
+"""Methods for loading models"""
+
+import os
+import tempfile
+from typing import Union
+
+import mlflow
+# pylint: disable=unused-import
+from torch import tensor  # used to eval tensors
+
+from meshnets.modules.model import MeshGraphNet
+from meshnets.modules.lightning_wrapper import MGNLightningWrapper
+from meshnets.utils.datasets import FromDiskGeometricDataset
+
+
+def load_model_mgn(tracking_uri: str, run_id: str, ckpt: Union[int, str],
+                   dataset: FromDiskGeometricDataset) -> MGNLightningWrapper:
```

Comment:
To make it explicit that this is a function to load a model from MLflow, I'd name it `load_mgn_model_from_mlfow(...)`.

Reply:
Done!